cypress/support/fragments/finance/funds/funds.js

```diff
@@ -115,10 +117,12 @@ export default {
     cy.xpath(viewTransactionsLinkXpath).click();
   },
 
-  checkTransaction: (value, fundCode) => {
-    // TODO: refactor using interactors (Mutli column list)
-    cy.expect(Pane({ id: transactionResultPaneId }).find(HTML(including('$' + value.toFixed(2)))).exists());
-    cy.expect(Section({ id: transactionResultPaneId }).find(HTML(including(fundCode))).exists());
+  checkTransaction: (rowNumber, transaction) => {
+    for (const value of Object.values(transaction)) {
```

Comment:
you can move Object.values(transaction) into new class and wrap it into getter. After this you will get ability to write something like transaction.getProperties().forEach() instead of for (const value of Object.values(transaction)) { with ability to expend logic in one place instead of several Object.values()

Reply:
Updated